src/extensions/default/QuickView/main.js

```diff
@@ -325,6 +325,11 @@ define(function (require, exports, module) {
         function hasLengthInPixels(args) {
             return (args.length > 1 && args[1].indexOf("px") > 0);
         }
+        
+        // Ensures that input is in usable hex format
+            function ensureHexFormat(str) {
```

Comment:
Whoops. That said, it passed my tests. And I had thought I had brackets-eslint installed, but I'm not sure anymore. Is that not the same as what would be found when running the command `grunt eslint`? I ask as it seems npm just downloaded something. If not, then it should be taken care of from here on out, at least.